interfaces/builtin/all.go

```diff
@@ -29,6 +29,7 @@ var allInterfaces = []interfaces.Interface{
 	&LocationControlInterface{},
 	&LocationObserveInterface{},
 	&NetworkManagerInterface{},
+	&ModemManagerInterface{},
```

Comment:
Totally minor nitpick, but it would be great since there are so many of these if they could be alphabetized.